client/src/components/UserProfile/UserBio.js

```diff
@@ -1,32 +1,31 @@
 import React from 'react'
 import { makeStyles } from '@material-ui/core/styles';
 import Button from '@material-ui/core/Button';
+import dayjs from 'dayjs'
 
 const useStyles = makeStyles(theme => ({
   button: {
     marginRight: '1em',
   }
 }));
 
-const UserBio = () => {
+const UserBio = (props) => {
 	const classes = useStyles();
-	// const { firstname, lastname, location, createdAt } = this.PaymentResponse;
-	// const fullname = `${firstname} ${lastname}`
+	const { firstName, lastName, location, createdAt, bio } = props.bio;
```

Comment:
inside props or props.bio?

Reply:
errors with just props or if i destructure it